docs/usage/configuration/config_documentation.md

```diff
@@ -462,6 +462,20 @@ See the docs [request log format](../administration/request_log.md).
 * `additional_resources`: Only valid for an 'http' listener. A map of
    additional endpoints which should be loaded via dynamic modules.
 
+Unix socket support (_Added in Synapse 1.87.0_):
```

Comment:
(Note to self to update version number before merge)

Reply:
I think this PR is ready to go (make this happen)

Reply:
Updated in [92c6c7b](https://github.com/matrix-org/synapse/pull/15708/commits/92c6c7b31de24e0174d12df241bd6bf713eb5786)